test/verify/check-machines

```diff
@@ -370,6 +380,71 @@ class TestMachines(MachineCase):
             print("Libvirt version does not support disk statistics")
             b.wait_present("#vm-{0}-disksstats-unsupported".format(name))
 
+    def testLibvirt(self):
+        b = self.browser
+        m = self.machine
+
+        libvirtServiceName = self.getLibvirtName()
+
+        def checkLibvirtEnabled():
+            try:
+                m.execute("systemctl -q is-enabled {0}".format(libvirtServiceName))
+                return True
+            except:  # exception if return code != 0
+                return False
+
+        self.startVm("subVmTest1")
+        self.login_and_go("/machines")
+
+        b.wait_in_text("body", "Virtual Machines")
+        b.wait_present("tbody tr th")
+        b.wait_in_text("tbody tr th", "subVmTest1")
+
+        b.wait(lambda: checkLibvirtEnabled())
```

Comment:
Why does this need to be waited for? On our test images it should already be enabled by default, so it would be weird if this failed the first few times.